engine/schema/src/main/java/com/cloud/upgrade/SystemVmTemplateRegistration.java

```diff
@@ -141,6 +146,14 @@ public SystemVmTemplateRegistration(String systemVmTemplateVersion) {
         this.systemVmTemplateVersion = systemVmTemplateVersion;
     }
 
+    public static String getMountCommand(String nfsVersion, String device, String dir) {
+        String cmd = "sudo mount -t nfs";
+        if (StringUtils.isNotBlank(nfsVersion)) {
+            cmd = String.format("%s -o vers=%s", cmd, nfsVersion);
+        }
+        return String.format("%s %s %s", cmd, device, dir);
+    }
+
```

Comment:
this looks like a good generic method for creating the mount command, but it is in `SystemVmTemplateRegistration`. Does it make sense to move it to a utility and search for other places to use this, @shwstppr ? (some deduplication may be resulting)